src/server.js

```diff
@@ -0,0 +1,24 @@
+const hapi = require('hapi');
+const inert = require('inert');
+const vision = require('vision');
+const routes = require('./routes');
+const path = require('path');
+
+const server = new hapi.Server();
+
+server.connection({
+  port: process.env.PORT || 8080,
+  routes: {
+    files: {
+      relativeTo: path.join(__dirname, '..', 'public')
```

Comment:
1. Am I right in saying this sets this as default config on all routes?
  2. The docs are super vague on what the files.relativeTo does.. Can you think of anything this would effect outside Inert?